diagnostic.md

```diff
@@ -38,7 +52,12 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
+$base-margin: 1.5em;
+$base-margin: 1.5em/2;
+
+.li {
```

Comment:
This will select classes named `li` and not the `li` element, which is selected with `li { ... }`.